nilearn/plotting/displays.py

```diff
@@ -957,7 +963,7 @@ def add_markers(self, marker_coords, marker_color='r', marker_size=30,
 
     def annotate(self, left_right=True, positions=True, scalebar=False,
                  size=12, scale_size=5.0, scale_units='cm', scale_loc=4,
-                 **kwargs):
+                 decimals=False, **kwargs):
```

Comment:
why not 0 rather than False

Reply:
0 should be better as you pointed out. I will update it as above.